### PR TITLE
Update README (close #369)

### LINKED ISSUE #369: Update README
The README file shows instructions for tracker testing using Vagrant.
We can suggest to test the tracker using Snowplow Micro instead.
It can be installed with docker and it's easy to setup.

Remove the section about Vagrant in the README.md file.
Add the section about SP Micro or SP Mini and related pages for a quick start with them.

### PR DESCRIPTION
**Pull request by AlexBenny**
Considering the README was quite obsolete I decided to keep that at minimum redirecting to the official doc website.
Some of the information here would be just duplicated or confusing.
A quick start on the README file can be redundant and much of the instructions would be basic knowledge for mobile developers or details specific for the package dependency manager.

For the future I would consider the README the place where to redirect to:
- official documentation - for instrumenting the tracker
- API documentation - for details on the API (maybe with a better rendering than the Doxygen default)
- contributing documentation - for setup of the project for contributing, architectural details and implementation guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,44 @@
 # Android Analytics for Snowplow
 
-[![actively-maintained]][tracker-classificiation] [![Build Status][gh-actions-image]][gh-actions] [![Coverage Status][coveralls-image]][coveralls] [![Release][release-image]][releases] [![License][license-image]][license]
+[![actively-maintained]][tracker-classificiation]
+[![Build Status][gh-actions-image]][gh-actions]
+[![Coverage Status][coveralls-image]][coveralls]
+[![Release][release-image]][releases]
```
Comment (paulboocock) [inline README.md:6]: `release-image` and `releases` seem to have vanished. I assume it should be:
https://img.shields.io/github/v/release/snowplow/snowplow-android-tracker

File: README.md [hunk 2]
```diff
@@ -1,63 +1,44 @@
...
-| ![i1][techdocs-image]          | ![i2][quick-start-image]       | ![i3][roadmap-image]   | ![i4][contributing-image]        |
-| **[Technical Docs][techdocs]** | **[Quick Start][setup]** | **[Roadmap][roadmap]** | **[Contributing](Contributing.md)** |
-
-Older documentation can be found [here][techdocs-old].
+| Contributing                                 |
+|----------------------------------------------|
+| [![i4][contributing-image]](CONTRIBUTING.md) |
+| [Contributing](CONTRIBUTING.md)              |
 
 ## Copyright and license
```
Comment (paulboocock) [inline README.md:43]: Update Copyright to 2021.

**Review by paulboocock — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -72,34 +53,33 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
-[techdocs-old]: https://github.com/snowplow/snowplow/wiki/Android-Tracker
-[setup]: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/android-tracker/android-1-4-0/#quick-start
-[roadmap]: https://github.com/snowplow/snowplow/wiki/Product-roadmap
+[coveralls]: https://coveralls.io/github/snowplow/snowplow-android-tracker?branch=master
+[coveralls-image]: https://coveralls.io/repos/github/snowplow/snowplow-android-tracker/badge.svg?branch=master
 
-[gh-actions]: https://github.com/snowplow/snowplow-android-tracker/actions
-[gh-actions-image]: https://github.com/snowplow/snowplow-android-tracker/workflows/build/badge.svg?branch=master
+[license]: https://www.apache.org/licenses/LICENSE-2.0
+[license-image]: https://img.shields.io/cocoapods/l/SnowplowTracker.svg
```
Comment (paulboocock) [inline README.md:69]: I've just spotted this sorry! Looks like this is using the Cocoapods license shield (which is a bit odd on Android), I think for consistency with other Snowplow repos, we should use the GitHub license: https://img.shields.io/github/license/snowplow/snowplow-android-tracker

**Review by paulboocock — APPROVED** [review]
LGTM bar my one last comment.

File: README.md (hunk 3 above)
Comment (AlexBenny) [reply README.md:69]: Ah damn, I've copied from the iOS repo one and I didn't noticed that one. Guess it's wrong there too. I'll use the Prepare for release commit to fix it on iOS. Also, I just noticed coveralls badge no longer work. Hope it's a momentary issue in their side.

Comment (paulboocock) [reply README.md:69]: Yeah I noticed the coveralls one, I assume its temporary. If it doesn't resolve itself we can fix it/remove it in v2.0.1 - it doesn't seem to work on Shields.io either https://img.shields.io/coveralls/github/snowplow/snowplow-android-tracker

**Pull request merged**